scripts/install_pio

```diff
@@ -2,7 +2,7 @@
 #type platformio > /dev/null 2>&1 && echo "pio is already installed" && exit
 # We install platformio in a virtualenv, because platformio and rosserial
 # depend on different versions of pyserial.
-sudo apt-get install -y python-virtualenv
+sudo pip install virtualenv
```

Comment:
Any particular reason for this change? Curious if it no longer works

Reply:
When building using docker, I was getting the following errors:

```
...
/home/pi/catkin_ws/src/openag_brain/scripts/install_pio: line 6: virtualenv: command not found
/home/pi/catkin_ws/src/openag_brain/scripts/install_pio: line 7: /home/pi/pio_env/bin/activate: No such file or directory
```

And there wasn't any straightforward error regarding the `sudo apt-get install virtualenv`, so perhaps it silently fails (given that the execution gets to line 6), or perhaps the binary is just not available in the PATH? Not sure to be honest, but installing it via pip worked consistently 💯

Perhaps we can [add `set -e`to `install_pio`](https://github.com/OpenAgInitiative/openag_brain/blob/develop/scripts/install_dev#L6-L8) as well, so if the command fails the whole execution halts.

Reply:
> Perhaps we can add set -eto install_pio as well, so if the command fails the whole execution halts.

Great idea.

> installing it via pip worked consistently 💯
I see. If @rbaynes can reproduce this it should be good to go.